src/app/onion/learning-object-builder/pages/info-page/info-page.component.html

```diff
@@ -18,7 +18,7 @@
         <hr />
         <clark-info-page-metadata>
           <div id="contributor-title" title>Who else worked on this?</div>
-          <div id="contributor-directions" directions>You can find users by their name, username, or email address.</div>
+          <div id="contributor-directions" directions>You can find users by their name, username, or email address. <b>Make sure to include yourself if you would like to be included in the object's attribution.</b></div>
```

Comment:
```suggestion
          <div id="contributor-directions" directions>You can find users by their name, username, or email address. <b>Make sure to include yourself as a contributor if you participated in the development of the materials for this Learning Object.</b></div>
```

Reply:
If you have a better suggestion feel free to play with this wording, but we can double check with Dr. Kaza too.

Reply:
No this seems fine to me.  Better worded.